File: app/src/main/java/org/stepic/droid/web/AssignmentResponse.java
package org.stepic.droid.web;

import org.stepik.android.model.Assignment;
import org.stepik.android.model.Meta;

import java.util.List;

public class AssignmentResponse extends MetaResponseBase {
    private final List<Assignment> assignments;

    public AssignmentResponse(Meta meta, List<Assignment> assignmentList) {
        super(meta);
        this.assignments = assignmentList;
    }

    public List<Assignment> getAssignments() {
        return assignments;
    }
}

File: app/src/main/java/org/stepik/android/remote/assignment/model/AssignmentResponse.kt
package org.stepik.android.remote.assignment.model

import com.google.gson.annotations.SerializedName
import org.stepik.android.model.Assignment
import org.stepik.android.model.Meta
import org.stepik.android.remote.base.model.MetaResponse

class AssignmentResponse(
    @SerializedName("meta")
    override val meta: Meta,
    @SerializedName("assignments")
    val assignments: List<Assignment>
) : MetaResponse
